exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/constants.py

```diff
@@ -14,3 +14,4 @@
 DD_ERROR_TYPE_TAG_KEY = "error.type"
 DD_ERROR_MSG_TAG_KEY = "error.msg"
 DD_ERROR_STACK_TAG_KEY = "error.stack"
+UNKNOWN_SERVICE_NAME = "unknown_service"
```

Comment:
```suggestion
_UNKNOWN_SERVICE_NAME = "unknown_service"
```

Reply:
removed this constant entirely but otherwise good catch